### ISSUE TITLE
[DocDB] PITR: UpdateConsensus RPC timed out when restoring a large number of tables

### ISSUE THREAD

**Issue by msun07** [yugabyte/yugabyte-db#15797]
Jira Link: [DB-5164](https://yugabyte.atlassian.net/browse/DB-5164)
### Description

UpdateConsensus RPC timed out when restoring a large number of tables, but both schema and data were restored successfully.

Log from master leader:
```
I0126 03:21:24.025527  2775 master_snapshot_coordinator.cc:500] Time spent Restore sys catalog took: real 149.561s      user 99.532s    sys 2.271s
I0126 03:21:24.034492  2775 snapshot_operation.cc:225] Forcing modify flushed frontier to 1.632283
I0126 03:21:24.038112  2775 db_impl.cc:5453] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [R]: [default] New memtable created with log file: #16
I0126 03:21:24.038221  3875 db_impl.cc:3337] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [R]: Calling FlushMemTableToOutputFile with column family [default], flush slots scheduled 1, total flush slots 1, compaction slots scheduled 0, compaction tasks [], total compaction slots 1
I0126 03:21:24.038275  3875 flush_job.cc:251] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [R]: [default] [JOB 897] Flushing memtable with next log file: 16
I0126 03:21:24.038300  3875 event_logger.cc:66] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [R]: EVENT_LOG_v1 {"time_micros": 1674703284038292, "job": 897, "event": "flush_started", "num_memtables": 1, "num_entries": 140714, "num_deletes": 0, "memory_usage": 47709184}
I0126 03:21:24.038311  3875 flush_job.cc:279] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [R]: [default] [JOB 897] Level-0 flush table #919: started
W0126 03:21:24.038329  2825 scoped_leader_shared_lock.cc:176] RPC took a long time (../../src/yb/master/catalog_manager_bg_tasks.cc:155, Run): 149.506s
    @     0x7f86480f4d7b  yb::master::CatalogManagerBgTasks::Run()
    @     0x7f864486f377  yb::Thread::SuperviseThread()
    @     0x7f8642e41694  start_thread
    @     0x7f8642b8341d  __clone
W0126 03:21:24.038246  2775 long_operation_tracker.cc:146] UpdateReplica took a long time: 149.575s
W0126 03:21:24.044943  2775 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:44377 => 127.0.0.2:7100 (request call id 3423350) took 149585ms (client timeout 3000ms).
I0126 03:21:24.049355 226180 raft_consensus.cc:620] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Fail of leader 21815b1f80f04fa9b5acbc3b1dc707e2 detected. Triggering leader pre-election, mode=NORMAL_ELECTION
I0126 03:21:24.049440 226180 raft_consensus.cc:3400] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Snoozing leader timeout detection for 3.086s
I0126 03:21:24.049486 226180 raft_consensus.cc:664] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Starting pre-election with config: opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } }
I0126 03:21:24.049738 226180 leader_election.cc:216] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 pre-election: Requesting vote from peer 21815b1f80f04fa9b5acbc3b1dc707e2
I0126 03:21:24.049839 226180 leader_election.cc:216] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 pre-election: Requesting vote from peer 1ccbe154605a4b3ab3952bd441f71819
W0126 03:21:24.049911  2870 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.3:45295 => 127.0.0.2:7100 (request call id 225554541) took 101691ms (client timeout 14998ms).
W0126 03:21:24.050038  2869 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.1:34255 => 127.0.0.2:7100 (request call id 228731240) took 101576ms (client timeout 15000ms).
W0126 03:21:24.049935  2873 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:47441 => 127.0.0.2:7100 (request call id 224387943) took 101690ms (client timeout 15000ms).
W0126 03:21:24.062250  2774 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:43973 => 127.0.0.2:7100 (request call id 224409202) took 85699ms (client timeout 15000ms).
W0126 03:21:24.073233  2874 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.3:57087 => 127.0.0.2:7100 (request call id 225575881) took 85709ms (client timeout 15000ms).
W0126 03:21:24.050012  2870 scoped_leader_shared_lock.cc:176] RPC took a long time (../../src/yb/master/master_cluster_service.cc:155, GetMasterRegistration): 101.691s
    @     0x7f8648103c7e  yb::master::ScopedLeaderSharedLock::~ScopedLeaderSharedLock()
    @     0x7f8648159e8b  yb::master::(anonymous namespace)::MasterClusterServiceImpl::GetMasterRegistration()
    @     0x7f8645567726  std::__1::__function::__func<>::operator()()
    @     0x7f8645565a2f  yb::master::MasterClusterIf::Handle()
    @     0x7f8644c7eb54  yb::rpc::ServicePoolImpl::Handle()
    @     0x7f8644c2739f  yb::rpc::InboundCall::InboundCallTask::Run()
    @     0x7f8644c8b10c  yb::rpc::(anonymous namespace)::Worker::Execute()
    @     0x7f864486f377  yb::Thread::SuperviseThread()
    @     0x7f8642e41694  start_thread
    @     0x7f8642b8341d  __clone
W0126 03:21:24.084398  2868 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.1:53383 => 127.0.0.2:7100 (request call id 228752425) took 85608ms (client timeout 15000ms).
W0126 03:21:24.087119  2773 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.3:50089 => 127.0.0.2:7100 (request call id 225597183) took 69709ms (client timeout 15000ms).
W0126 03:21:24.065017 226183 scoped_leader_shared_lock.cc:176] RPC took a long time (../../src/yb/master/catalog_manager.cc:12088, RebuildYQLSystemPartitions): 145.066s
    @     0x7f8648103c7e  yb::master::ScopedLeaderSharedLock::~ScopedLeaderSharedLock()
    @     0x7f8648085d9d  yb::master::CatalogManager::RebuildYQLSystemPartitions()
    @     0x7f8644879ef6  yb::ThreadPool::DispatchThread()
    @     0x7f864486f377  yb::Thread::SuperviseThread()
    @     0x7f8642e41694  start_thread
    @     0x7f8642b8341d  __clone
W0126 03:21:24.099685  2867 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:44081 => 127.0.0.2:7100 (request call id 224430459) took 69721ms (client timeout 15000ms).
W0126 03:21:24.101476  2866 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.1:50715 => 127.0.0.2:7100 (request call id 228773673) took 69622ms (client timeout 15000ms).
W0126 03:21:24.101547  2872 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:51617 => 127.0.0.2:7100 (request call id 224451737) took 53729ms (client timeout 15000ms).
W0126 03:21:24.101575  2871 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.3:55999 => 127.0.0.2:7100 (request call id 225618468) took 53724ms (client timeout 15000ms).
W0126 03:21:24.101675  2726 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.1:42711 => 127.0.0.2:7100 (request call id 228794893) took 53616ms (client timeout 15000ms).
W0126 03:21:24.101744  2864 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:37327 => 127.0.0.2:7100 (request call id 224465419) took 43377ms (client timeout 15000ms).
W0126 03:21:24.101922  2728 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.1:36117 => 127.0.0.2:7100 (request call id 228808492) took 43376ms (client timeout 15000ms).
I0126 03:21:24.102428  2787 raft_consensus.cc:2989] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]:  Leader pre-election vote request: Denying vote to candidate 1ccbe154605a4b3ab3952bd441f71819 for term 2 because replica is already servicing an update from a current leader or another vote.
I0126 03:21:24.102515  2787 leader_election.cc:367] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 pre-election: Vote granted by peer 1ccbe154605a4b3ab3952bd441f71819
I0126 03:21:24.102558  2787 leader_election.cc:242] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 pre-election: Election decided. Result: candidate won.
W0126 03:21:24.102092 226340 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:51799 => 127.0.0.2:7100 (request call id 224486685) took 27380ms (client timeout 15000ms).
W0126 03:21:24.102387 226395 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.1:56887 => 127.0.0.2:7100 (request call id 228851014) took 11369ms (client timeout 15000ms).
I0126 03:21:24.102727 226180 raft_consensus.cc:3400] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Snoozing leader timeout detection for 3.162s
I0126 03:21:24.102751 226180 raft_consensus.cc:3293] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Leader pre-election won for term 2
I0126 03:21:24.102762 226180 raft_consensus.cc:620] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Fail of leader 21815b1f80f04fa9b5acbc3b1dc707e2 detected. Triggering leader election, mode=NORMAL_ELECTION
I0126 03:21:24.102773 226180 raft_consensus.cc:3400] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Snoozing leader timeout detection for 3.094s
I0126 03:21:24.102784 226180 raft_consensus.cc:3461] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]: Advancing to term 2
W0126 03:21:24.102270 226345 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.1:37673 => 127.0.0.2:7100 (request call id 228829751) took 27368ms (client timeout 15000ms).
W0126 03:21:24.102344 226394 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.3:35599 => 127.0.0.2:7100 (request call id 225674993) took 11371ms (client timeout 15000ms).
W0126 03:21:24.101893 226316 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.3:58569 => 127.0.0.2:7100 (request call id 225632289) took 43376ms (client timeout 15000ms).
W0126 03:21:24.102294 226392 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:41961 => 127.0.0.2:7100 (request call id 224507945) took 11379ms (client timeout 15000ms).
W0126 03:21:24.102144 226342 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.3:53683 => 127.0.0.2:7100 (request call id 225653646) took 27377ms (client timeout 15000ms).
W0126 03:21:24.099741  2867 scoped_leader_shared_lock.cc:176] RPC took a long time (../../src/yb/master/master_cluster_service.cc:155, GetMasterRegistration): 69.721s
    @     0x7f8648103c7e  yb::master::ScopedLeaderSharedLock::~ScopedLeaderSharedLock()
    @     0x7f8648159e8b  yb::master::(anonymous namespace)::MasterClusterServiceImpl::GetMasterRegistration()
    @     0x7f8645567726  std::__1::__function::__func<>::operator()()
    @     0x7f8645565a2f  yb::master::MasterClusterIf::Handle()
    @     0x7f8644c7eb54  yb::rpc::ServicePoolImpl::Handle()
    @     0x7f8644c2739f  yb::rpc::InboundCall::InboundCallTask::Run()
    @     0x7f8644c8b10c  yb::rpc::(anonymous namespace)::Worker::Execute()
    @     0x7f864486f377  yb::Thread::SuperviseThread()
    @     0x7f8642e41694  start_thread
    @     0x7f8642b8341d  __clone
I0126 03:21:24.194124 226180 consensus_meta.cc:317] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433: Updating active role from FOLLOWER to FOLLOWER. Consensus state: current_term: 2 leader_uuid: "" config { opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } }, has_pending_config = 0
I0126 03:21:24.194204 226180 raft_consensus.cc:664] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 FOLLOWER]: Starting election with config: opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } }
I0126 03:21:24.197300 226180 leader_election.cc:216] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 election: Requesting vote from peer 21815b1f80f04fa9b5acbc3b1dc707e2
I0126 03:21:24.197355 226180 leader_election.cc:216] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 election: Requesting vote from peer 1ccbe154605a4b3ab3952bd441f71819
W0126 03:21:24.197592  2787 leader_election.cc:277] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 election: RPC error from VoteRequest() call to peer 21815b1f80f04fa9b5acbc3b1dc707e2: Network error (yb/util/net/socket.cc:540): recvmsg error: Connection refused (system error 111)
I0126 03:21:24.231381  2775 leader_election.cc:367] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 election: Vote granted by peer 1ccbe154605a4b3ab3952bd441f71819
I0126 03:21:24.231416  2775 leader_election.cc:242] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [CANDIDATE]: Term 2 election: Election decided. Result: candidate won.
I0126 03:21:24.231454 226180 raft_consensus.cc:3400] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 FOLLOWER]: Snoozing leader timeout detection for 3.065s
I0126 03:21:24.231487 226180 raft_consensus.cc:3307] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 FOLLOWER]: Leader election won for term 2
I0126 03:21:24.231513 226180 replica_state.cc:1416] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 FOLLOWER]: SetLeaderNoOpCommittedUnlocked(0), committed: 1.632283, received: 1.632283
I0126 03:21:24.231545 226180 consensus_meta.cc:317] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433: Updating active role from FOLLOWER to LEADER. Consensus state: current_term: 2 leader_uuid: "be60a930428e432b9fccb9e5f16f3433" config { opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } }, has_pending_config = 0
I0126 03:21:24.231596 226180 raft_consensus.cc:3346] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 LEADER]: Calling mark dirty synchronously for reason code NEW_LEADER_ELECTED
I0126 03:21:24.231611 226180 sys_catalog.cc:418] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [sys.catalog]: SysCatalogTable state changed. Locked=1. Reason: New leader be60a930428e432b9fccb9e5f16f3433 elected. Latest consensus state: current_term: 2 leader_uuid: "be60a930428e432b9fccb9e5f16f3433" config { opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } }
I0126 03:21:24.231679 226180 sys_catalog.cc:422] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [sys.catalog]: This master's current role is: LEADER
W0126 03:21:24.125349 226342 scoped_leader_shared_lock.cc:176] RPC took a long time (../../src/yb/master/master_cluster_service.cc:155, GetMasterRegistration): 27.387s
    @     0x7f8648103c7e  yb::master::ScopedLeaderSharedLock::~ScopedLeaderSharedLock()
    @     0x7f8648159e8b  yb::master::(anonymous namespace)::MasterClusterServiceImpl::GetMasterRegistration()
    @     0x7f8645567726  std::__1::__function::__func<>::operator()()
    @     0x7f8645565a2f  yb::master::MasterClusterIf::Handle()
    @     0x7f8644c7eb54  yb::rpc::ServicePoolImpl::Handle()
    @     0x7f8644c2739f  yb::rpc::InboundCall::InboundCallTask::Run()
    @     0x7f8644c8b10c  yb::rpc::(anonymous namespace)::Worker::Execute()
    @     0x7f864486f377  yb::Thread::SuperviseThread()
    @     0x7f8642e41694  start_thread
    @     0x7f8642b8341d  __clone
I0126 03:21:24.242444 226180 raft_consensus.cc:1056] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 LEADER]: Becoming Leader. State: Replica: be60a930428e432b9fccb9e5f16f3433, State: 1, Role: LEADER, Watermarks: {Received: 1.632283 Committed: 1.632283} Leader: 0.0
I0126 03:21:24.242548 226180 consensus_queue.cc:268] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [LEADER]: Queue going to LEADER mode. State: All replicated op: 0.0, Majority replicated op: 1.632283, Committed index: 1.632283, Last applied: 1.632283, Last appended: 1.632283, Current term: 2, Majority size: 2, State: QUEUE_OPEN, Mode: LEADER, active raft config: opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } }
I0126 03:21:24.242727 226180 raft_consensus.cc:1074] Sending NO_OP at op 1.632283
I0126 03:21:24.244865  3080 client-internal.cc:2314] New master addresses: [127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100, 127.0.0.1:7100, 127.0.0.2:7100, 127.0.0.3:7100]
W0126 03:21:24.245884 226180 raft_consensus.cc:1371] Leader lease expiration was not set: Illegal state (yb/consensus/replica_state.cc:1347): NoOp of current term is not committed (majority_replicated_lease_expiration_ = <uninitialized>, majority_replicated_ht_lease_expiration_ = 0)
W0126 03:21:24.245942 226180 replica_state.cc:794] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 LEADER]: Can't advance the committed index across term boundaries until operations from the current term are replicated. Last committed operation was: 1.632283, New majority replicated is: 1.632283, Current term is: 2
I0126 03:21:24.246032  2726 master_cluster_service.cc:164] Patching role from leader to follower because of: Leader not ready to serve requests (yb/consensus/consensus.cc:166): Leader not yet replicated NoOp to be ready to serve requests
W0126 03:21:24.246034  2787 consensus_peers.cc:538] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 -> Peer 21815b1f80f04fa9b5acbc3b1dc707e2 ([host: "127.0.0.1" port: 7100], []): Couldn't send request.  Status: Network error (yb/util/net/socket.cc:540): recvmsg error: Connection refused (system error 111). Retrying in the next heartbeat period. Already tried 1 times. State: 2
I0126 03:21:24.246176  2787 consensus_queue.cc:1310] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [LEADER]: Connected to new peer: { peer: 1ccbe154605a4b3ab3952bd441f71819 is_new: 0 last_received: 1.632283 next_index: 632284 last_known_committed_idx: 632283 is_last_exchange_successful: 0 needs_remote_bootstrap: 0 member_type: VOTER num_sst_files: 0 last_applied: 1.632283 }
W0126 03:21:24.250262 226405 raft_consensus.cc:1371] Leader lease expiration was not set: Illegal state (yb/consensus/replica_state.cc:1347): NoOp of current term is not committed (majority_replicated_lease_expiration_ = <uninitialized>, majority_replicated_ht_lease_expiration_ = 0)
I0126 03:21:24.250313 226405 replica_state.cc:1416] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 LEADER]: SetLeaderNoOpCommittedUnlocked(1), committed: 1.632283, received: 2.632284
I0126 03:21:24.250335 226405 replica_state.cc:785] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 2 LEADER]: Advanced the committed_op_id across terms. Last committed operation was: 1.632283 New committed index is: 2.632284
I0126 03:21:24.251924 226404 catalog_manager.cc:1038] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433: Loading table and tablet metadata into memory for term 2
I0126 03:21:24.251966 226404 catalog_manager.cc:1128] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433: VisitSysCatalog: Wait on leader_lock_ for any existing operations to finish. Term: 2
I0126 03:21:24.252071 226404 catalog_manager.cc:1139] T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433: VisitSysCatalog: Acquire catalog manager lock_ before loading sys catalog.
```

Log from master follower:
```
I0126 03:18:57.430253 226179 raft_consensus.cc:1027] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 1 FOLLOWER]: ReportFailDetected: Starting NORMAL_ELECTION...
I0126 03:18:58.556720 20807 restore_sys_catalog_state.cc:789] PITR: Will remove table: 00004000000030008000000000058e08
I0126 03:18:59.506522 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f868ea228 -> pg_constraint_conparentid_index: total: 129, updates: 128, inserts: 0, deletes: 1
I0126 03:18:59.572407 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f869a04e8 -> pg_attrdef: total: 20, updates: 10, inserts: 0, deletes: 10
W0126 03:19:00.704967  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:44345 => 127.0.0.3:7100 (request call id 3423352) took 3002ms (client timeout 3000ms).
W0126 03:19:03.781445  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:44333 => 127.0.0.3:7100 (request call id 3423353) took 3003ms (client timeout 3000ms).
W0126 03:19:07.035344  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:40055 => 127.0.0.3:7100 (request call id 3423355) took 3000ms (client timeout 3000ms).
I0126 03:19:08.372567 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f873d0f28 -> pg_constraint: total: 2451, updates: 2432, inserts: 0, deletes: 19
W0126 03:19:10.135205  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:34573 => 127.0.0.3:7100 (request call id 3423357) took 3000ms (client timeout 3000ms).
W0126 03:19:13.449208  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33245 => 127.0.0.3:7100 (request call id 3423359) took 3002ms (client timeout 3000ms).
W0126 03:19:16.707023  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33975 => 127.0.0.3:7100 (request call id 3423361) took 3001ms (client timeout 3000ms).
W0126 03:19:19.904093  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:35461 => 127.0.0.3:7100 (request call id 3423363) took 3000ms (client timeout 3000ms).
W0126 03:19:22.964689  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33099 => 127.0.0.3:7100 (request call id 3423365) took 3001ms (client timeout 3000ms).
I0126 03:19:26.101063 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f8cccf8e8 -> pg_depend: total: 7680, updates: 7592, inserts: 0, deletes: 88
W0126 03:19:26.120759  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:44345 => 127.0.0.3:7100 (request call id 3423367) took 3003ms (client timeout 3000ms).
W0126 03:19:29.587222  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:44333 => 127.0.0.3:7100 (request call id 3423369) took 3001ms (client timeout 3000ms).
I0126 03:19:31.494427 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f86a028a8 -> pg_index: total: 2340, updates: 2322, inserts: 0, deletes: 18
I0126 03:19:31.596328 20807 restore_sys_catalog_state.cc:187] PITR: Incrementing pg_yb_catalog version to 7
I0126 03:19:31.625860 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f8cce5468 -> pg_attrdef_adrelid_adnum_index: total: 8, updates: 4, inserts: 0, deletes: 4
W0126 03:19:32.860610  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:40055 => 127.0.0.3:7100 (request call id 3423371) took 3000ms (client timeout 3000ms).
W0126 03:19:35.999269  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:34573 => 127.0.0.3:7100 (request call id 3423373) took 3001ms (client timeout 3000ms).
I0126 03:19:37.841347 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f7d5ed7a8 -> pg_attribute_relid_attnam_index: total: 2738, updates: 2688, inserts: 0, deletes: 50
I0126 03:19:39.425324 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f8cce5268 -> pg_class_relname_nsp_index: total: 554, updates: 548, inserts: 0, deletes: 6
W0126 03:19:39.559780  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33245 => 127.0.0.3:7100 (request call id 3423376) took 3004ms (client timeout 3000ms).
I0126 03:19:39.876132 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f7d6ace68 -> pg_constraint_conname_nsp_index: total: 129, updates: 128, inserts: 0, deletes: 1
I0126 03:19:40.563596 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f7d6df628 -> pg_constraint_conrelid_contypid_conname_index: total: 258, updates: 256, inserts: 0, deletes: 2
I0126 03:19:40.923321 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f869a14e8 -> pg_constraint_contypid_index: total: 129, updates: 128, inserts: 0, deletes: 1
W0126 03:19:42.709504  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33975 => 127.0.0.3:7100 (request call id 3423378) took 3001ms (client timeout 3000ms).
I0126 03:19:44.079281 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f8cca76e8 -> pg_depend_depender_index: total: 960, updates: 949, inserts: 0, deletes: 11
W0126 03:19:45.888942  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:35461 => 127.0.0.3:7100 (request call id 3423380) took 3002ms (client timeout 3000ms).
I0126 03:19:47.023488 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f8695bb28 -> pg_depend_reference_index: total: 960, updates: 949, inserts: 0, deletes: 11
I0126 03:19:47.391327 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f868974e8 -> pg_index_indrelid_index: total: 130, updates: 129, inserts: 0, deletes: 1
I0126 03:19:48.151391 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f86800928 -> pg_class_tblspc_relfilenode_index: total: 277, updates: 274, inserts: 0, deletes: 3
W0126 03:19:49.127182  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33099 => 127.0.0.3:7100 (request call id 3423382) took 3001ms (client timeout 3000ms).
I0126 03:19:49.763979 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f869e70a8 -> pg_type_typname_nsp_index: total: 568, updates: 562, inserts: 0, deletes: 6
I0126 03:19:49.764519 20807 restore_sys_catalog_state.cc:964] PITR: Pg system table 0x0000559f869ca2a8 -> pg_sequence: total: 80, updates: 72, inserts: 0, deletes: 8
W0126 03:19:52.352547  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:44345 => 127.0.0.3:7100 (request call id 3423384) took 3000ms (client timeout 3000ms).
W0126 03:19:55.482877  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:44333 => 127.0.0.3:7100 (request call id 3423385) took 3001ms (client timeout 3000ms).
W0126 03:19:58.815169  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:40055 => 127.0.0.3:7100 (request call id 3423387) took 3000ms (client timeout 3000ms).
W0126 03:20:02.249511  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:34573 => 127.0.0.3:7100 (request call id 3423390) took 3008ms (client timeout 3000ms).
W0126 03:20:05.302877  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33245 => 127.0.0.3:7100 (request call id 3423392) took 3000ms (client timeout 3000ms).
W0126 03:20:08.710533  2798 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33975 => 127.0.0.3:7100 (request call id 3423394) took 3001ms (client timeout 3000ms).
I0126 03:21:22.437178 226179 raft_consensus.cc:620] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 1 FOLLOWER]: Fail of leader 21815b1f80f04fa9b5acbc3b1dc707e2 detected. Triggering leader pre-election, mode=NORMAL_ELECTION
I0126 03:21:22.437254 226179 raft_consensus.cc:3400] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 1 FOLLOWER]: Snoozing leader timeout detection for 3.100s
I0126 03:21:22.437278 226179 raft_consensus.cc:664] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 1 FOLLOWER]: Starting pre-election with config: opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } }
I0126 03:21:22.437480 226179 leader_election.cc:216] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [CANDIDATE]: Term 2 pre-election: Requesting vote from peer 21815b1f80f04fa9b5acbc3b1dc707e2
I0126 03:21:22.437577 226179 leader_election.cc:216] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [CANDIDATE]: Term 2 pre-election: Requesting vote from peer be60a930428e432b9fccb9e5f16f3433
W0126 03:21:22.438192  2798 leader_election.cc:277] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [CANDIDATE]: Term 2 pre-election: RPC error from VoteRequest() call to peer 21815b1f80f04fa9b5acbc3b1dc707e2: Network error (yb/util/net/socket.cc:540): recvmsg error: Connection refused (system error 111)
W0126 03:21:22.433256  2826 scoped_leader_shared_lock.cc:176] RPC took a long time (../../src/yb/master/catalog_manager_bg_tasks.cc:155, Run): 147.881s
    @     0x7f27a5289d7b  yb::master::CatalogManagerBgTasks::Run()
    @     0x7f27a1a04377  yb::Thread::SuperviseThread()
    @     0x7f279ffd6694  start_thread
    @     0x7f279fd1841d  __clone
W0126 03:21:22.441291  2885 yb_rpc.cc:336] Call yb.master.MasterService.GetMasterRegistration 127.0.0.2:60017 => 127.0.0.3:7100 (request call id 224387944) took 100082ms (client timeout 15000ms).
W0126 03:21:22.450350 20807 yb_rpc.cc:336] Call yb.consensus.ConsensusService.UpdateConsensus 127.0.0.1:33099 => 127.0.0.3:7100 (request call id 3423349) took 148030ms (client timeout 3000ms).
W0126 03:21:22.439764 226181 scoped_leader_shared_lock.cc:176] RPC took a long time (../../src/yb/master/catalog_manager.cc:12088, RebuildYQLSystemPartitions): 144.756s
    @     0x7f27a5298c7e  yb::master::ScopedLeaderSharedLock::~ScopedLeaderSharedLock()
    @     0x7f27a521ad9d  yb::master::CatalogManager::RebuildYQLSystemPartitions()
    @     0x7f27a1a0eef6  yb::ThreadPool::DispatchThread()
    @     0x7f27a1a04377  yb::Thread::SuperviseThread()
    @     0x7f279ffd6694  start_thread
    @     0x7f279fd1841d  __clone
I0126 03:21:22.987459  3876 flush_job.cc:311] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [R]: [default] [JOB 897] Level-0 flush table #919: 7469957 bytes OK frontiers: { smallest: { op_id: 1.632282 hybrid_time: { physical: 1674702107399198 } history_cutoff: <invalid> hybrid_time_filter: <invalid> max_value_level_ttl_expiration_time: <invalid> primary_schema_version: <NULL> cotable_schema_versions: [] } largest: { op_id: 1.632283 hybrid_time: { physical: 1674703133844955 } history_cutoff: <invalid> hybrid_time_filter: <invalid> max_value_level_ttl_expiration_time: <initial> primary_schema_version: <NULL> cotable_schema_versions: [] } }
I0126 03:21:22.987522  3876 event_logger.cc:66] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [R]: EVENT_LOG_v1 {"time_micros": 1674703282987500, "cf_name": "default", "job": 897, "event": "table_file_creation", "file_number": 919, "file_size": 7469957, "table_properties": {"data_size": 7458390, "data_index_size": 32329, "filter_size": 0, "filter_index_size": 0, "raw_key_size": 12334975, "raw_average_key_size": 61, "raw_value_size": 37191208, "raw_average_value_size": 184, "num_data_blocks": 532, "num_entries": 201785, "num_filter_blocks": 0, "num_data_index_blocks": 1, "filter_policy_name": "", "kDeletedKeys": "0"}}
I0126 03:21:22.988960  3876 db_impl.cc:1992] [default] Level summary: files[8] max score 1.60
I0126 03:21:22.989076  3876 memtable_list.cc:385] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [R]: [default] Level-0 commit table #919 started
I0126 03:21:22.989091  3876 memtable_list.cc:401] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [R]: [default] Level-0 commit table #919: memtable #1 done
I0126 03:21:22.989101  3876 event_logger.cc:76] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [R]: EVENT_LOG_v1 {"time_micros": 1674703282988919, "job": 897, "event": "flush_finished", "lsm_state": [8]}
I0126 03:21:24.050626  2798 raft_consensus.cc:2366] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 1 FOLLOWER]: Pre-election. Granting vote for candidate be60a930428e432b9fccb9e5f16f3433 in term 2
I0126 03:21:24.114459 20807 leader_election.cc:381] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [CANDIDATE]: Term 2 pre-election: Vote denied by peer be60a930428e432b9fccb9e5f16f3433. Message: Service unavailable (yb/consensus/raft_consensus.cc:2990): T 00000000000000000000000000000000 P be60a930428e432b9fccb9e5f16f3433 [term 1 FOLLOWER]:  Leader pre-election vote request: Denying vote to candidate 1ccbe154605a4b3ab3952bd441f71819 for term 2 because replica is already servicing an update from a current leader or another vote.
I0126 03:21:24.114511 20807 leader_election.cc:242] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [CANDIDATE]: Term 2 pre-election: Election decided. Result: candidate lost.
I0126 03:21:24.203060  2798 raft_consensus.cc:3461] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 1 FOLLOWER]: Advancing to term 2
I0126 03:21:24.219230  2798 consensus_meta.cc:317] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819: Updating active role from FOLLOWER to FOLLOWER. Consensus state: current_term: 2 leader_uuid: "" config { opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } }, has_pending_config = 0
I0126 03:21:24.219326  2798 raft_consensus.cc:3400] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]: Snoozing leader timeout detection for 3.284s
W0126 03:21:24.219422 20807 outbound_call.cc:364] RPC callback for RPC call yb.consensus.ConsensusService.RequestConsensusVote -> { remote: 127.0.0.2:7100 idx: 2 protocol: 0x00007f27a1e47c60 -> tcpc } , state=FINISHED_SUCCESS. took 0.105s
I0126 03:21:24.229302  2798 raft_consensus.cc:3012] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]:  Leader election vote request: Granting yes vote for candidate be60a930428e432b9fccb9e5f16f3433 in term 2.
I0126 03:21:24.229682 226403 raft_consensus.cc:3400] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]: Snoozing leader timeout detection for 3.027s
I0126 03:21:24.229732 226403 raft_consensus.cc:3246] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]: Leader Pre-election lost for term 2. Reason: None given. Originator:
I0126 03:21:24.245501 20807 raft_consensus.cc:1722] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]: Refusing update from remote peer be60a930428e432b9fccb9e5f16f3433: Log matching property violated. Preceding OpId in replica: 1.632283. Preceding OpId from leader: 2.632284. (index mismatch)
I0126 03:21:24.245537 20807 raft_consensus.cc:1938] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]: Returning from UpdateConsensus because of error: error { code: PRECEDING_ENTRY_DIDNT_MATCH status { code: ILLEGAL_STATE message: "Log matching property violated. Preceding OpId in replica: 1.632283. Preceding OpId from leader: 2.632284. (index mismatch)" source_file: "../../src/yb/consensus/raft_consensus.cc" source_line: 1720 errors: "00" } }
I0126 03:21:24.246812  2798 consensus_meta.cc:317] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819: Updating active role from FOLLOWER to FOLLOWER. Consensus state: current_term: 2 leader_uuid: "be60a930428e432b9fccb9e5f16f3433" config { opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } }, has_pending_config = 0
I0126 03:21:24.246887  2798 raft_consensus.cc:3346] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]: Calling mark dirty synchronously for reason code NEW_LEADER_ELECTED
I0126 03:21:24.246907  2798 sys_catalog.cc:418] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [sys.catalog]: SysCatalogTable state changed. Locked=1. Reason: New leader be60a930428e432b9fccb9e5f16f3433 elected. Latest consensus state: current_term: 2 leader_uuid: "be60a930428e432b9fccb9e5f16f3433" config { opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } }
I0126 03:21:24.246946  2798 sys_catalog.cc:422] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [sys.catalog]: This master's current role is: FOLLOWER
I0126 03:21:24.247056  3059 client-internal.cc:2314] New master addresses: [127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100, 127.0.0.1:7100, 127.0.0.2:7100, 127.0.0.3:7100]
I0126 03:21:24.251436 20807 raft_consensus.cc:3346] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [term 2 FOLLOWER]: Calling mark dirty synchronously for reason code FOLLOWER_NO_OP_COMPLETE
I0126 03:21:24.251474 20807 sys_catalog.cc:418] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [sys.catalog]: SysCatalogTable state changed. Locked=1. Reason: Replicate of NO_OP complete on follower. Latest consensus state: current_term: 2 leader_uuid: "be60a930428e432b9fccb9e5f16f3433" config { opid_index: -1 peers { permanent_uuid: "21815b1f80f04fa9b5acbc3b1dc707e2" member_type: VOTER last_known_private_addr { host: "127.0.0.1" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "be60a930428e432b9fccb9e5f16f3433" member_type: VOTER last_known_private_addr { host: "127.0.0.2" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } peers { permanent_uuid: "1ccbe154605a4b3ab3952bd441f71819" member_type: VOTER last_known_private_addr { host: "127.0.0.3" port: 7100 } cloud_info { placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" } } }
I0126 03:21:24.251520 20807 sys_catalog.cc:422] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819 [sys.catalog]: This master's current role is: FOLLOWER
W0126 03:21:24.251549 20807 mvcc.cc:435] T 00000000000000000000000000000000 P 1ccbe154605a4b3ab3952bd441f71819: Received propagated safe time { physical: 1674702107399198 } less than the old value: { physical: 1674703134419599 }. This could happen on followers when a new leader is elected.
I0126 03:21:24.665228  2778 tcp_stream.cc:322] { local: 127.0.0.3:42165 remote: 127.0.0.1:7100 }:  Recv failed: Network error (yb/util/net/socket.cc:540): recvmsg error: Connection refused (system error 111)
I0126 03:21:25.736531  2778 tcp_stream.cc:322] { local: 127.0.0.3:36751 remote: 127.0.0.1:7100 }:  Recv failed: Network error (yb/util/net/socket.cc:540): recvmsg error: Connection refused (system error 111)
I0126 03:22:30.651229  2778 reactor.cc:467] Master_R000: DEBUG: Closing idle connection: Connection (0x0000559f6b3927f8) server 127.0.0.3:43975 => 127.0.0.3:7100 - it has been idle for 65.0988s
I0126 03:22:30.651333  2778 reactor.cc:467] Master_R000: DEBUG: Closing idle connection: Connection (0x0000559f6b392378) server 127.0.0.3:35491 => 127.0.0.3:7100 - it has been idle for 65.0988s
```

[DB-5164]: https://yugabyte.atlassian.net/browse/DB-5164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by rthallamko3** [comment]
@msun07 , Do you have steps to repro the problem?

**Comment by msun07** [comment]
This is part of the scalability test. @rthallamko3

Repro steps:
1. Create rf=3 cluster
2. Create a database with a large number of tables (e.g., Temenos schemas, 60K tables)
3. Create a snapshot schedule on the database
4. Perform a PITR restore

**Comment by rthallamko3** [comment]
Thanks @msun07 , Do you have script or schema handy for which it repro'd in your case? That will help speed up the investigation.

**Comment by druzac** [comment]
I saw some similar log chatter in #15202:
```
W0313 10:45:01.162500 18045 consensus_peers.cc:544] T 00000000000000000000000000000000 P 31e1754844d740d3b5f8db839d63f429 -> Peer 5d34874a016f44fdb3b002b6472b2e18 ([host: "10.9.72.169" port: 7100], []): Couldn't send request.  Status: Timed out (yb/rpc/outbound_call.cc:489): UpdateConsensus RPC (request call id 79395) to 10.9.72.169:7100 timed out after 3.000s. Retrying in the next heartbeat period. Already tried 1 times. State: 2
```

Given we have a flaky system test that reproduces these timeouts and the ticket is older, I'm going to close this and dedupe to #15202.

**Comment by druzac** [comment]
Duplicate of #15202.
